packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats-description.tsx

```diff
@@ -1,12 +1,22 @@
 import React from 'react';
 import { ActionSheet, Text } from '@deriv-com/quill-ui';
 import { Localize } from '@deriv/translations';
+import { DESCRIPTION_VIDEO_ID, UNIFIED_MODE_VIDEO_ID } from 'Modules/Trading/Helpers/video-config';
 
 const AccumulatorStatsDescription = ({ onActionSheetClose }: { onActionSheetClose: () => void }) => {
     return (
         <ActionSheet.Portal showHandlebar={false}>
             <div className='stats-description'>
-                <div className='stats-description__placeholder' />
+                <div className='stats-description__player-wrapper'>
+                    <div className='stats-description__player-wrapper__player'>
+                        <iframe
```

Comment:
we can't use VideoPlayer component for this?

Reply:
I think VideoPlayer component contains a lot of unnecessary for this case logic, like functions for controls and checking if it should autoplaying or not.

Reply:
why dont we use video tag ?

Reply:
Nope , it doesn't work with VideoPlayer, I tried but it doesn't handles the other options to make it a gif like look

Reply:
video tags doesn't support the format that we get from cloudflare .